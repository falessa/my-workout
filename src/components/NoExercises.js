import React from 'react';
import { StyleSheet, View, Text } from 'react-native';

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: '#fff',
        alignItems: 'center',
        justifyContent: 'center',
    }
})

const NoExercises = () => {
    return (
        
        <View style={styles.container}>
            <Text>No exercises added yet</Text>
        </View>
        
    )
}

export default NoExercises