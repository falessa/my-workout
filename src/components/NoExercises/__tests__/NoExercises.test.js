import "react-native";
import React from "react";
import { create } from "react-test-renderer";
import NoExercises from '../NoExercises';

describe('NoExercises tests', () => {
    const props = {
        day: 2
    }

    const wrapper = create(<NoExercises {...props} />)

    test('Render NoExercises', () => {
        expect(wrapper).toMatchSnapshot()
    })
})