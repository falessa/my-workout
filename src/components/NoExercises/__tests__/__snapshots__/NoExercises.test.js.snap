// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoExercises tests Render NoExercises 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    noExercisesAddedForDay
     
    2
  </Text>
</View>
`;
